src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -270,7 +270,11 @@ export const ContentMessageComponent = ({
 
         <div css={deliveredMessageIndicator}>
           {showDeliveredMessageIcon && (
-            <div data-uie-name="status-message-read-receipt-delivered" title={t('conversationMessageDelivered')}>
+            <div
+              data-uie-name="status-message-read-receipt-delivered"
+              title={showDeliveredMessageIcon ? t('conversationMessageDelivered') : ''}
+              data-uie-value={showDeliveredMessageIcon ? 'Delivered' : ''}
```

Comment:
I feel this `value` is not adding any extra info. If the icon is there it already means that the message has been delivered